packages/core/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import MedlyLogoName from './assets/MedlyLogo.svg';
+import MedlyLogo from './assets/MLogo.svg';
+import { Container } from './Logo.styled';
+import { LogoProps } from './types';
+
+const Logo: React.FC<LogoProps> = ({ showName, logo: LogoEl, name: LogoName }) => (
+    <Container>
+        <LogoEl />
```

Comment:
You don't have to call the component. You can just write like this `{logo}`. The benefit of this is if they want to pass any prop to component the can do it.